components/centraldashboard/app/api_workgroup.ts

```diff
@@ -163,13 +165,7 @@ export class WorkgroupApi {
      * Retrieves all namespaces in case of basic auth.
      */
     async getAllWorkgroups(fakeUser: string): Promise<SimpleBinding[]> {
-        let bindings;
-        try{
-            //try catch is for a retry when it fails
```

Comment:
This is our change, it should be left in.

Reply:
Change added back